File: Modul 7/Soal 5/PRAK605-2210817220018-Piola Evania.c
#include <stdio.h>

int main(void)
{
    for(int i = 1; i < 3; i++){
        int b, isi, total = 0, hasil[10][10];
        scanf("%d", &b);
        int m1[b][b], m2[b][b];
        printf("\nMatriks A\n");
        for(int i = 0; i < b; i++){
            for(int j = 0; j < b; j++){
                scanf("%d", &isi);
                m1[i][j] = isi;
            }
        }
        printf("\nMatriks B\n");
         for(int i = 0; i < b; i++){
            for(int j = 0; j < b; j++){
                scanf("%d", &isi);
                m2[i][j] = isi;
            }
        }
         for(int i = 0; i < b; i++){
            for(int j = 0; j < b; j++){
                for(int k = 0; k < b; k++){
                    total += m1[i][k] * m2[k][j];
                }
                hasil[i][j] = total;
                total = 0;
            }
         }
        printf("\nMatriks AXB\n");
        for(int i = 0; i < b; i++){
            for(int j = 0; j < b; j++){
                    printf("%d ", hasil[i][j]);
            }
            printf("\n");
        }
        printf("\n\n");
    }
}

File: Modul 7/Soal 5/PRAK605-2210817220018-Piola Evania.py
for i in range(2): 
    b = int(input())
    m1 = []
    print("Matriks A")
    for i in range(0, b):
        isi1 = list(map(int, input().split()))
        m1.append(isi1[0 : b])
    m2 = []
    print("Matriks B")
    for i in range(0, b):
        isi2 = list(map(int, input().split()))
        m2.append(isi2[0 : b])
    print("\nMatriks AXB\n")
    for i in range(0, b):
        for j in range(0, b):
            hasil = 0
            for k in range(0, b):
                hasil += m1[i][k] * m2[k][j]
            print(hasil, end=' ')
        print()
    print("\n")